CMakeLists.txt

```diff
@@ -351,7 +351,7 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
   SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
   GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/slicersalt/${extension_name}
-  GIT_TAG        aecbfec228ac095d75ada0066a96eb2c128a5f65 # slicersalt-2019-11-20-bd4bdcf
+  GIT_TAG        e33344cb73286da278320a8902d97a4b5f4054b7 # slicersalt-2019-11-20-bd4bdcf
```

Comment:
Can you please update the comment with the date for the slicer tag?

Reply:
The date is the name of the branch.  That date shouldn't change because the update is a backport - the base date of the slicer branch is the same.

Reply:
Gotcha, thanks!